protobuf/protoc-gen-cruxclient/api_generator.cc

```diff
@@ -417,7 +417,7 @@ void APIGenerator::PrintDjinniYAML(
     printer->Print("objc:\n");
     printer->Indent();
     printer->Print(vars, "typename: '$objc_class_prefix$$message_name$'\n");
-    printer->Print(vars, "header: '\"$dir$/$objc_file_name$.pbobjc.h\"'\n");
+    printer->Print(vars, "header: '<CruxDomain/$dir$_$objc_file_name$.pbobjc.h>'\n");
```

Comment:
Should it be uppercase `CruxDomain` or just `crux`?

Reply:
The crux is used by legacy crux framework, and `CruxDomain` is more aligned with Apple style framework name

Reply:
Does this change mean that the `s12-proto` is implicitly coupled to the new Crux engine/domain?

Reply:
This generated files are for Crux Domain APIs. but yes. It's coupled. Let me try if I can get away with just use `S12` and having `CruxDomain` depends on it.

Reply:
@jimmyhoran ok, it doesn't work with `S12`

<img width="1130" alt="Screen Shot 2020-02-10 at 9 12 18 am" src="https://user-images.githubusercontent.com/1888415/74111628-488f4200-4bea-11ea-85d7-e2c08d1463fa.png">

Reply:
Did dropping the moulds import work?

Reply:
Yep